Contour.m

```diff
@@ -4,14 +4,13 @@
 
     properties
         frequencies % An array of raw frequency values
-        attributes  % A ContourAttributes instance
+        tempres % Sample rate of frequency
     end
 
     methods
         function cont = Contour(freq)
-            % Construct a contour instance from the given frequency array
-            cont.frequencies = freq;
-            cont.attributes  = ContourAttributes(freq);
+            cont.frequencies = freq.frequency; % frequency component of csv
+            cont.tempres  = freq.tempres; % tempres componenent of the csv
```

Comment:
freq will already be an array of frequencies, not a CSV file. The constructor should instead (after our talk) take a frequency array and the tempres directly.